### DISCUSSION TITLE
Video Tutorials

### DISCUSSION THREAD

**Discussion by SalitaNgDiyos** [rubiot/ibiblia#84] [Show and tell]
**1. Short introduction to iBiblia:** https://youtu.be/QXdI9f4Osfo

**2. **Copying Strong's and Morphology codes**:** https://youtu.be/s92qOu_aWZM

**3. **How to Setup a Translation Project****: https://youtu.be/f8NubvjkgRs

**Comment by rubiot** [comment]
Thanks a lot, brother!,

**Reply by SalitaNgDiyos** [reply]
Just one of my tiny ways to support your work, bro. May God's grace help us perfect the work He has started in us!

**Reply by rubiot** [reply]
Amen!

**Comment by jonathangkoehn** [comment]
Thanks for making these Salita, I am watching them.

**Reply by SalitaNgDiyos** [reply]
Hi Jonathan,
Hope it helps a bit. Hope someone can improve on the videos for English speaking people.

In Christ,
Edmund

**Comment by rubiot** [comment]
It was very useful to see how someone other then me uses the app. It helped me make it a little bit sturdier. Since then I've added a number of checks to prevent some errors you've encountered.
